build/ng-admin-only.min.css

```diff
@@ -2,4 +2,437 @@
\ No newline at end of file
+ */
+body {
```

Comment:
Minification has disappeared, but it will come back in another PR, with other build optimizations.